ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceFileManager.swift

```diff
@@ -92,15 +92,16 @@ public class ResourceFileManager {
       try fileManager.removeItem(at: destination)
     }
 
-    // Throws an error if the destination file already exists, and there's no
-    // built-in override parameter.  Hence, the previous if-block.
-    if source.startAccessingSecurityScopedResource() {
-      defer { source.stopAccessingSecurityScopedResource() }  // The Swift version of 'finally'.
+    // If we've been provided a security-scoped resource URL,
+    // it needs special handling.  This function needs to accept
+    // both scoped & non-scoped URLs.
+    if source.startAccessingSecurityScopedResource() { // only succeeds if scoped
+      // The Swift version of 'finally'.
```

Comment:
We don't really need comments about language-specific keywords -- expectation is that we learn how to use the tooling 😁  (this may not hold for some API or framework related comments, especially if there are non-obvious or less documented aspects to them)

Reply:
Well, considering that I'm the only dev on the team that works with Swift right now, and it's quite distinct from the usual keyword for this role, I figured it was merited.  Makes team cross-language reviews easier.

It's nowhere near as pervasive as Swift's optionals, so it's much easier to forget.